### PR TITLE
Always register the derived CC from the reg profile ##anal

### PR DESCRIPTION
**Pull request by trufae**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md#code-style)
- [x] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [radare2 book](https://github.com/radareorg/radare2book) with the relevant information (if needed)

**Detailed description**

Some archs lack any definition of calling conventions. In addition, r2 can start without cc defined when running without proper installation or just using the single static binary. This patch registers the derived regprofile right after setting the arch

**Test plan**

this will probably break some tests,so its worth reviewing and updating them to include the new reg profile

**Closing issues**

...

### CODE REVIEW

File: libr/core/cbin.c [hunk 1]
```diff
@@ -751,6 +752,10 @@ R_API void r_core_anal_cc_init(RCore *core) {
 		sdb_concat_by_path (core->anal->sdb_cc, dbpath);
 		core->anal->sdb_cc->path = strdup (dbpath);
 	}
+	if (sdb_isempty (core->anal->sdb_cc)) {
+		eprintf ("Warning: Missing calling conventions for '%s' / '%s'. Deriving it from the regprofile.\n", anal_arch, asm_cpu);
+	}
+	r_core_cmd0 (core, "tcc `arcc`");
```
Comment (thestr4ng3r) [inline libr/core/cbin.c:758]: Are you sure this will never cmd-inject itself? I would rather use api here.

**Review by thestr4ng3r — CHANGES_REQUESTED** [review]

**Comment by thestr4ng3r** [comment]
lgtm when green

**Comment by trufae** [comment]
I think the warning gets printed several times in some cases. Maybe its better to remove it. Wait a bit

**Comment by trufae** [comment]
It's green. can we merge this? (the red is because the http test)

**Pull request merged**